app/javascript/viewer/uv_manager.js

```diff
@@ -111,6 +126,15 @@ export default class UVManager {
     this.cdlTimer.initializeTimer()
   }
 
+  createClover () {
+    this.uvElement.show()
+    const uvElement = document.getElementById('uv')
+    const root = ReactDOM.createRoot(uvElement)
+    // Setting canvasHeight is a workaround until this issue is resolved:
+    // https://github.com/samvera-labs/clover-iiif/issues/179
+    root.render(React.createElement(Viewer, { iiifContent: this.manifest, options: { informationPanel: { open: false, renderToggle: false }, background: 'white', withCredentials: true, showTitle: false, showIIIFBadge: false, canvasHeight: uvElement.clientHeight } }))
```

Comment:
This workaround is nice, but it breaks in the case that we have two videos - the video selector is pushed below the fold. :(

Reply:
Work around improved.